README.md

```diff
@@ -1,5 +1,8 @@
 # flutter_architecture_samples
 
+[![Build Status](https://travis-ci.org/mmcc007/flutter_architecture_samples.svg?branch=master)](https://travis-ci.org/mmcc007/flutter_architecture_samples)
```

Comment:
I've enabled Travis and Coveralls for this project. Could you please update these links to point to this repo instead of `mmcc007`?